examples/decoding/plot_receptive_field.py

```diff
@@ -151,4 +158,94 @@
 ax.set(title="Topomap of model coefficients\nfor delay %s" % time_plot)
 mne.viz.tight_layout()
 
+
+###############################################################################
+# Create and fit a stimulus reconstruction model (a.k.a. backward model)
+# ----------------------------------------------------------------------
+#
+# We will now construct a model to find the linear relationship between the
+# speech signal and a time-delayed version of the EEG. This can be advantageous
+# as we exploit all of the available neural data in a multivariate context,
+# compared to the forward case which treats each M/EEG channel as an
+# independent feature. Therefore, low SNR stimuli such as speech will likely
+# have better backward models.
+
+# We use the same lags as in [1]_
```

Comment:
no need for the underscore here...this won't be rendered as rST so the linking won't happen